lib/config/validation.ts

```diff
@@ -1,4 +1,5 @@
 import is from '@sindresorhus/is';
+import isUrl from 'is-url';
```

Comment:
Can [is.urlString](https://github.com/sindresorhus/is#urlstringvalue) work instead?

Reply:
Done, thanks.